Cannot upgrade to 0.38.0: count_rows_ipc not found in this scope

### Checks

- [X] I have checked that this issue has not already been reported.
- [X] I have confirmed this bug exists on the [latest version](https://crates.io/crates/polars) of Polars.

### Reproducible example

```bash
$ cargo build
...
   Compiling polars-row v0.38.0
   Compiling polars-json v0.38.0
   Compiling polars-parquet v0.38.0
   Compiling polars-time v0.38.0
   Compiling polars-io v0.38.0
error[E0425]: cannot find function `count_rows_ipc` in this scope
  --> /Users/davidr/.cargo/registry/src/index.crates.io-6f17d22bba15001f/polars-plan-0.38.0/src/logical_plan/functions/count.rs:49:21
   |
49 |                     count_rows_ipc(&mut reader)
   |                     ^^^^^^^^^^^^^^ not found in this scope

For more information about this error, try `rustc --explain E0425`.
error: could not compile `polars-plan` (lib) due to 1 previous error
```


### Log output

_No response_

### Issue description

Upgrading a Rust project to Polars 0.38.0 fails because polars-plan fails to compile.

### Expected behavior

All Polars modules build based on their current Polars dependencies.

### Installed versions

Upgrading from 0.37.0 to 0.38.0.

--- a/crates/polars-pipe/src/operators/chunks.rs
+++ b/crates/polars-pipe/src/operators/chunks.rs
@@ -103,6 +103,7 @@ impl StreamingVstacker {
     }
 }
 
+#[cfg(any(feature = "parquet", feature = "ipc", feature = "csv"))]
 impl Default for StreamingVstacker {
     /// 4 MB was chosen based on some empirical experiments that showed it to
     /// be decently faster than lower or higher values, and it's small enough

--- a/crates/polars-pipe/src/operators/chunks.rs
+++ b/crates/polars-pipe/src/operators/chunks.rs
@@ -44,7 +44,7 @@ pub(crate) fn chunks_to_df_unchecked(chunks: Vec<DataChunk>) -> DataFrame {
 ///
 /// Changing the `DataFrame` into contiguous chunks is the caller's
 /// responsibility.
-#[cfg(feature = "parquet")]
+#[cfg(any(feature = "parquet", feature = "ipc", feature = "csv"))]
 #[derive(Clone)]
 pub(crate) struct StreamingVstacker {
     current_dataframe: Option<DataFrame>,
@@ -52,7 +52,7 @@ pub(crate) struct StreamingVstacker {
     output_chunk_size: usize,
 }
 
-#[cfg(feature = "parquet")]
+#[cfg(any(feature = "parquet", feature = "ipc", feature = "csv"))]
 impl StreamingVstacker {
     /// Create a new instance.
     pub fn new(output_chunk_size: usize) -> Self {
@@ -103,7 +103,6 @@ impl StreamingVstacker {
     }
 }
 
-#[cfg(feature = "parquet")]
 impl Default for StreamingVstacker {
     /// 4 MB was chosen based on some empirical experiments that showed it to
     /// be decently faster than lower or higher values, and it's small enough
@@ -114,7 +113,7 @@ impl Default for StreamingVstacker {
 }
 
 #[cfg(test)]
-#[cfg(feature = "parquet")]
+#[cfg(any(feature = "parquet", feature = "ipc", feature = "csv"))]
 mod test {
     use super::*;
 

--- a/crates/polars-plan/src/logical_plan/functions/count.rs
+++ b/crates/polars-plan/src/logical_plan/functions/count.rs
@@ -1,4 +1,4 @@
-#[cfg(feature = "parquet")]
+#[cfg(feature = "ipc")]
 use arrow::io::ipc::read::get_row_count as count_rows_ipc;
 #[cfg(feature = "parquet")]
 use polars_io::cloud::CloudOptions;

